docs/sql-migration-guide.md

```diff
@@ -201,6 +201,20 @@ license: |
         </tr>
     </table>
 
+  - Since Spark 3.0, special values are supported in conversion from strings to dates and timestamps. Those values are simply notational shorthands that will be converted to ordinary date or timestamp values when read. The following string values are supported for dates:
+    - `epoch [zoneId]` - 1970-01-01
+    - `today [zoneId]` - the current date in the time zone specified by `spark.sql.session.timeZone`.
+    - `yesterday [zoneId]` - the current date - 1
+    - `tomorrow [zoneId]` - the current date + 1
+    - `now` - the date of running the current query. It has the same notion as today.
+  For example `SELECT date 'tomorrow' - date 'yesterday';` should output `2`. Here are special timestamp values:
+    - `epoch [zoneId]` - 1970-01-01 00:00:00+00 (Unix system time zero)
+    - `today [zoneId]` - midnight today.
```

Comment:
Shall we remove the `.` at the end?

Reply:
It seems need to put `.` at the end of every list: http://site.uit.no/english/writing-style/bulletlist/

Reply:
Maybe separate each item by `;` and `.` at the end of the last sentence?

Reply:
I removed all dots because some items don't form complete sentences, and others must not as well for consistency .